resources/js/Components/Form/CharacterCreateForm.test.js

```diff
@@ -9,7 +8,7 @@ const factions = [
     'The Harpers',
     'The Order of the Gauntlet',
     'The Emerald Enclave',
-    'The Lords\' Alliance',
+    'The Lords Alliance',
```

Comment:
This should not be changed. The faction name is correct. [Source](https://dnd.wizards.com/dungeons-and-dragons/story/factions)

Reply:
See above.